services/freecodecamp/freecodecamp-points.service.js

```diff
@@ -62,18 +57,18 @@ export default class FreeCodeCampPoints extends BaseJsonService {
           username,
         },
       },
+      httpErrors: { 404: 'profile not found' },
     })
   }
 
   static transform(response, username) {
     const { entities } = response
 
-    if (entities === undefined)
```

Comment:
If we're removing this handling, we tighten up the schema and make sure the values we need are now required in the schema. At the moment the schema is written to accept bad values so we could handle the not found case like this.

Reply:
Good catch, thanks!